commands/user_e2e_test.go

```diff
@@ -116,6 +116,7 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 
 	s.RunForAllClients("Search for an existing user", func(c client.Client) {
 		printer.Clean()
+		printer.SetFormat(printer.FormatJSON)
```

Comment:
Had to add line this because this test breaks due to the `printer.SetFormat()` on line 19 of `channel_e2e_test.go`